eas.json

```diff
@@ -15,6 +15,7 @@
       }
     },
     "production": {
+      "distribution": "internal",
```

Comment:
I think this is not the intended use of internal distribution. According to https://docs.expo.dev/build/internal-distribution/:

> Uploading your app to TestFlight and Google Play beta can be time-consuming (...) and limiting (...). Both Android and iOS provide alternative mechanisms to distribute apps directly to testers, so they can download and install them to physical devices directly from a web browser as soon as the builds are completed.

> EAS Build can help you with this by providing shareable URLs for your builds with instructions on how to get them running, so you can share a single URL with a teammate that'll include all of the information they need to test the app. Interestingly, the example they use is of a `preview` profile, nothing related to production. 

There's an alternative and more explicit way to generate an APK. Taken https://docs.expo.dev/build-reference/apk/#configuring-a-profile-to-build-apks, we can add the following in this production profile.

```json
"android": {
   "buildType": "apk"
}
```

For the record, from https://docs.expo.dev/build/automating-submissions/, it seems that the process is rather meant to be handled automatically by using the `--auto-submit` flag. We're not there yet. I wouldn't spend time making it work now.

Reply:
We'll need to work on this later on. https://docs.expo.dev/build/eas-json/#production-builds is pretty explicit on how this should be done:

> Production builds must be installed through their respective app stores; they cannot be installed directly on your Android Emulator or device, or iOS Simulator or device. The only exception to this is if you explicitly set "buildType": "apk" for Android on your build profile; however, **it is recommended to use AAB when submitting to stores, and this is the default configuration.**

Reply:
for the record, I solved this using `buildType` in https://github.com/coopdevs/timeoverflow-mobile-app/pull/15.